SOURCES/0028-xenopsd-set-xen-platform-pci-bar-uc-key-in-xenstore.patch

```diff
@@ -0,0 +1,80 @@
+From bc9580e4abb422d47167a2fdf95b9fe2f78e8bc6 Mon Sep 17 00:00:00 2001
+From: Anthoine Bourgeois <anthoine.bourgeois@vates.tech>
+Date: Fri, 27 Jun 2025 10:48:26 +0200
+Subject: [PATCH] xenopsd: set xen-platform-pci-bar-uc key in xenstore
+
+This patch add a new parameter named 'xen-platform-pci-bar-uc' in
+xenopsd config file who has a default value of 'true' to keep the
+default behavior of hvmloader.  Putting 'false' to this parameter will
+tell xenopsd to add a xenstore key of '0' in:
+'/local/domain/<domid>/hvmloader/pci/xen-platform-pci-bar-uc'.
+Only this key set to 0 will change the behavior of hvmloader.
+
+This changeset is link to this xen commit:
+https://xenbits.xen.org/gitweb/?p=xen.git;a=commit;h=22650d6054625be10172fe0c78b9cadd1a39bd63
+
+Signed-off-by: Anthoine Bourgeois <anthoine.bourgeois@vates.tech>
+---
+ ocaml/xenopsd/lib/xenopsd.ml | 7 +++++++
+ ocaml/xenopsd/xc/domain.ml   | 2 ++
+ ocaml/xenopsd/xenopsd.conf   | 6 ++++++
+ 3 files changed, 15 insertions(+)
+
+diff --git a/ocaml/xenopsd/lib/xenopsd.ml b/ocaml/xenopsd/lib/xenopsd.ml
+index 9c5e83e04..c38efa67f 100644
+--- a/ocaml/xenopsd/lib/xenopsd.ml
++++ b/ocaml/xenopsd/lib/xenopsd.ml
+@@ -49,6 +49,8 @@ let default_vbd_backend_kind = ref "vbd"
+ 
+ let ca_140252_workaround = ref false
+ 
++let xen_platform_pci_bar_uc = ref true
```

Comment:
We can probably set the ref to "0" and always write it to xenstore I think. And we can later switch to "1". I personally prefer things that are explicit when possible so always have a value that is either "0" or "1". But it is just a suggestion and I'm ok with the current patch.